test/specs/scale.time.tests.js

```diff
@@ -1081,8 +1081,8 @@ describe('Time scale tests', function() {
 
 				options.time.unit = 'year';
 				chart.data.labels = [];
-				var meta = chart.getDatasetMeta(1);
-				meta.hidden = true;
+				chart.getDatasetMeta(0).hidden = true;
```

Comment:
What labels does it generate? Is it actually a bug not clearing the timestamps for labels?

Reply:
The issue isn't to do with the labels, but rather it's that `source: 'data'` and we previously had a visible dataset. I'm not sure why it would be expected to generate no ticks when you have no labels and one visible dataset.

Reply:
Oh, right. Was just curious how it would plot the one visible dataset with no labels to get the x-coordinate from.
I mean it should have no timestamps to work with.
And I was thinking this could be a bug in the parsing, not detecting change of labels and re-parsing.

Reply:
Tried to answer my own question, but the test does not fail if these changes are removed.

Reply:
I notice it's marked as "requested changes". Are you saying you think the test should go back to the way it was before? It didn't really make sense to me that way.

Reply:
Yes, that's what I was saying. You need to also remove the `correctly handle empty data.labels` from `it` or revert the change. I'd revert for not reducing coverage.

Reply:
Ok. I've reverted the change